karapace/rapu.py

```diff
@@ -393,14 +393,12 @@ async def _handle_request(
             )
             headers = {"Content-Type": "application/json"}
             resp = aiohttp.web.Response(body=body, status=status.value, headers=headers)
-        except (ConnectionError, aiohttp.ClientError) as connection_error:
-            # TCP level connection errors, e.g. TCP reset, client closes connection.
-            self.log.debug("Connection error.", exc_info=connection_error)
+        except (ConnectionError, aiohttp.ClientError, asyncio.CancelledError, asyncio.TimeoutError) as exc:
+            # TCP level connection errors and timeouts, e.g. TCP reset, client closes connection, task takes too long.
+            error_msg = "Unexpected connection or timeout error"
+            self.log.debug(error_msg, exc_info=exc)
```

Comment:
maybe error log?

Reply:
`error` would go to sentry, as this is handled, we can bump to info if you want, but i'd leave as is for CI costs. Prometheus would handle these error metrics once we've added them.